### PR TITLE
WIP: Feature/add button component

### PR DESCRIPTION
**Pull request by larsdouweschuitema**

### CODE REVIEW

**Review by dine — COMMENTED** [review]
Since we have semantic release it will only change the version if you use the proper commit messages format: https://github.com/semantic-release/semantic-release#commit-message-format

**Review by bartderijk — APPROVED** [review]

**Comment by jpsc** [comment]
> Since we have semantic release it will only change the version if you use the proper commit messages format: https://github.com/semantic-release/semantic-release#commit-message-format

Yes, but it's also possible to create that proper message with a squash commit before merging.

**Review by jpsc — CHANGES_REQUESTED** [review]
I would try to rebase from master and figure out what we can take from this PR that improves the existing button component.

File: src/components/ui-button.vue [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    },
+    methods: {
+        handleClick(e) {
+            this.$emit('handleClick');
+        }
+    }
+  };
+</script>
+
+<style lang="scss" scoped>
```
Comment (sukalas-a) [inline src/components/ui-button.vue:36]: You can now use Tailwind and refactor all these styles

**Comment by larsdouweschuitema** [comment]
> I would try to rebase from master and figure out what we can take from this PR that improves the existing button component.
That's the plan. Use what you have already made and what can be taken from this. Will keep you posted.

**Pull request closed without merging**